commit 63917f0b5ba2a932d4fca7f67d1a1eae9034269e
Merge: d822d2a1e331 aa4a73a0a23a
Author: Catalin Marinas <catalin.marinas@arm.com>
Date:   Wed Jun 12 16:48:38 2013 +0100

    Merge branch 'kvm-arm64/kvm-for-3.11' of git://git.kernel.org/pub/scm/linux/kernel/git/maz/arm-platforms into upstream
    
    * 'kvm-arm64/kvm-for-3.11' of git://git.kernel.org/pub/scm/linux/kernel/git/maz/arm-platforms: (33 commits)
      arm64: KVM: document kernel object mappings in HYP
      arm64: KVM: MAINTAINERS update
      arm64: KVM: userspace API documentation
      arm64: KVM: enable initialization of a 32bit vcpu
      arm64: KVM: 32bit guest fault injection
      arm64: KVM: 32bit specific register world switch
      arm64: KVM: CPU specific 32bit coprocessor access
      arm64: KVM: 32bit handling of coprocessor traps
      arm64: KVM: 32bit conditional execution emulation
      arm64: KVM: 32bit GP register access
      arm64: KVM: define 32bit specific registers
      arm64: KVM: Build system integration
      arm64: KVM: PSCI implementation
      arm64: KVM: Plug the arch timer
      ARM: KVM: timer: allow DT matching for ARMv8 cores
      arm64: KVM: Plug the VGIC
      arm64: KVM: Exit handling
      arm64: KVM: HYP mode world switch implementation
      arm64: KVM: hypervisor initialization code
      arm64: KVM: guest one-reg interface
      ...
    
    Conflicts:
            arch/arm64/Makefile

diff --cc arch/arm64/Makefile
index 79dd13dc8837,d0ce30a275e0..741f04fd636a
--- a/arch/arm64/Makefile
+++ b/arch/arm64/Makefile
@@@ -37,7 -37,7 +37,8 @@@ TEXT_OFFSET := 0x0008000
  export	TEXT_OFFSET GZFLAGS
  
  core-y		+= arch/arm64/kernel/ arch/arm64/mm/
- core-$(CONFIG_XEN)		+= arch/arm64/xen/
+ core-$(CONFIG_KVM) += arch/arm64/kvm/
++core-$(CONFIG_XEN) += arch/arm64/xen/
  libs-y		:= arch/arm64/lib/ $(libs-y)
  libs-y		+= $(LIBGCC)